packages/flutter_tools/lib/executable.dart

```diff
@@ -186,7 +195,12 @@ Future<Null> main(List<String> args) async {
 }
 
 Future<File> _createCrashReport(List<String> args, dynamic error, Chain chain) async {
```

Comment:
Rename this to `_createLocalCrashReport()` now to distinguish it from what `CrashReportSender` does.